packages/dev-server/.eslintrc.cjs

```diff
@@ -1,5 +1,14 @@
 module.exports = {
   extends: '@callstack/eslint-config/node',
+  parserOptions: {
+    project: true,
+    tsconfigRootDir: __dirname,
+  },
+  rules: {
+    /* These 2 rules do not play nicely with fastify */
+    'require-await': 0,
+    '@typescript-eslint/no-floating-promises': 0,
+  },
```

Comment:
as the comment mentions, lot's of Fastify API uses `thenables` which incorrectly get picked up as `Promise`, that's why these 2 rules were disabled